### ISSUE TITLE
[Feature Request] Is it possible to add volume and brightness slider like any other video players out there.

### ISSUE THREAD

**Issue by t0m5krt** [crackededed/Xtra#362]
Title.
![Screenshot_20230909-114303](https://github.com/crackededed/Xtra/assets/91048191/a3164ec2-4459-406f-acb1-4cac1dee11f9)
![Screenshot_20230909-114317](https://github.com/crackededed/Xtra/assets/91048191/f2270345-bd4d-4ca9-9aeb-99a4b436d3c1)

Screens from NewPipe

**Comment by thinkpad4** [comment]
Doesn't your phone have both of those features built in?

**Comment by t0m5krt** [comment]
> Doesn't your phone have both of those features built in?

I mean, why bother going to settings if you can adjust it in the video player with one just swipe.

**Comment by thinkpad4** [comment]
I don't get why you need to adjust the brightness when watching Twitch?  And every phone has volume buttons on the side.

**Comment by hellishvictor** [comment]
On the bottom left corner of the Xtra player there's a headphone icon, and clicking on it will shows a slider for the volume.

**Comment by exalented** [comment]
> I don't get why you need to adjust the brightness when watching Twitch? And every phone has volume buttons on the side.

Uhhh, the stream is dark. I believe at least the brightness controls should be added as a toggle.

**Comment by Palloxin** [comment]
any considerations of this after 3 years? It is REALLY a fundamental feature. If you try it for a bit u cannot go back.

**Comment by Palloxin** [comment]
Volume buttons are a chore compared to the slider, I am not a fan of Brightness tho. For me Volume alone is enough

**Comment by Palloxin** [comment]
@crackededed look it is PEAK (slider feature)

https://github.com/user-attachments/assets/95b1d1b1-d9a9-40c4-9c57-b17557c8dadd

**Comment by Teddy131** [comment]
Please dont add this. It is super annoying accidently changing brightness or volume. If it gets implementedy this feature needs the ability to be turned off completely

**Comment by Palloxin** [comment]
Yes, a setting to decide if to put it or not. It IS A VERY GOOD FEATURE
